supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -41,6 +41,15 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
     JesJobExecutionActor.props(jobDescriptor, jesConfiguration, initializationData.toJes.get, serviceRegistryActor).withDispatcher(BackendDispatcher)
   }
 
+  override def cacheHitCopyingActorProps(jobDescriptor: BackendJobDescriptor,
+                                         initializationData: Option[BackendInitializationData],
+                                         serviceRegistryActor: ActorRef,
+                                         cachedJobOutputs: JobOutputs): Props = {
+    // The `get` here is safe.  If the initialization data wasn't built, the workflow execution wouldn't have advanced to the
+    // point of trying to create job cache hit copying actors.
```

Comment:
That's fine if you can show that the failure will manifest itself upstream in a sane way